File: Python/1009_bitwiseComplement.py
class Solution(object):
    def bitwiseComplement(self, n):
        num = 1
        while n >= num*2:
            num*=2
        count = 0
        while num != 0:
            if num - n > 0:
                count+=num
            else:
                n -= num
            num/=2
        return count

        
        

File: C++/1009_bitwiseComplement.cpp
class Solution {
public:
    int bitwiseComplement(int n) {
        int num = 1;
        while(n >= num*2)
            num*=2;
        int count = 0;
        while(num != 0){
            if(num - n > 0)
                count+=num;
            else
                n -= num;
            num/=2;
        }
        return count;
    }
};
